datus/tools/mcp_tools/tool_metadata_extractor.py

```
@@ -0,0 +1,447 @@
+
+        Args:
+            tool_name: Name of the tool
+            mcp_server: MCP server name
+
+        Returns:
+            Detailed tool metadata
+        """
+        # Try to get tool schema or description
+        try:
+            # Attempt to call the tool with no arguments to get parameter info
+            success, result, data = await self.mcp_manager.call_tool(
```

Ruff (0.14.5)

Unpacked variable `data` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)